git-squash-tags

```diff
@@ -29,7 +31,7 @@ if [ $? -ne 0 ]; then # non-zero exit code indicates branch does not exist
   git checkout $tags_first
   git symbolic-ref HEAD refs/heads/$branch_squash
   rm .git/index
-  tag=$tags_first; commitAll
+  previous=$tags_first; tag=$tags_first; commitAll
```

Comment:
previous is used in the first commitAll so that it contains no content from the log. Maybe leaving it empty is a better approach.